exampleplugins/5.2/SwagCustomProductBoxLayout/plugin.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<plugin xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:noNamespaceSchemaLocation="../../../engine/Shopware/Components/Plugin/schema/plugin.xsd">
+    <label lang="de">SwagCustomProductBoxLayout</label>
+    <label lang="en">SwagCustomProductBoxLayout</label>
+    <version>1.0.0</version>
+    <copyright>(c) by Shopware AG</copyright>
+    <license>proprietary</license>
+    <link>http://store.shopware.com</link>
+    <author>Shopware AG</author>
+
+    <compatibility minVersion="5.2.0" />
```

Comment:
ok, did not see the min version :smile: 

than if you set this up, you can remove the build method from the base class

Reply:
I updated the minVersion to 5.2.10 and removed the build method.